src/System.Memory/src/System/Buffers/MemoryPool.cs

```diff
@@ -0,0 +1,49 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Buffers
+{
+    /// <summary>
+    /// Represents a pool of memory blocks.
+    /// </summary>
+    public abstract class MemoryPool<T> : IDisposable
+    {
+        /// <summary>
+        /// Returns a singleton instance of a MemoryPool based on arrays.
+        /// </summary>
+        public static MemoryPool<T> Shared => ArrayMemoryPool<T>.Shared;
+
+        /// <summary>
+        /// Returns a memory block capable of holding at least <paramref name="minBufferSize">minBufferSize</paramref> elements of T.
```

Comment:
`Returns a memory block capable of holding at least <paramref name="minBufferSize"/> elements of T.`?